src/Search/FlagrowApi.php

```diff
@@ -11,13 +11,7 @@
 use Illuminate\Support\Arr;
 use Psr\Http\Message\ResponseInterface;
 
-/**
- * Class FlagrowApi
- * @package Flagrow\Bazaar\Search
- *
- * @info Contextually binding the Guzzle client wasn't working. Something was very off, this works though.
- */
-class FlagrowApi extends Client
+final class FlagrowApi extends Client
```

Comment:
The class cannot be extended. I did this on some of our classes to prevent other extensions from re-using it's logic or even doing easy calls to our api.